## Forest Fire Data Exploration in Indonesia from 2000 till 2018

In [1]:
# Directory
setwd("../../")

setwd("Data/ESRI/Extracted_yearly_IND")

In [2]:
# Load libraries
library(sf)
library(tidyverse)
library(DT)
library(shiny)

Linking to GEOS 3.11.0, GDAL 3.5.3, PROJ 9.1.0; sf_use_s2() is TRUE

-- Attaching core tidyverse packages ------------------------ tidyverse 2.0.0 --
v dplyr     1.1.4     v readr     2.1.5
v forcats   1.0.0     v stringr   1.5.1
v ggplot2   3.5.1     v tibble    3.2.1
v lubridate 1.9.4     v tidyr     1.3.1
v purrr     1.0.2     
-- Conflicts ------------------------------------------ tidyverse_conflicts() --
x dplyr::filter() masks stats::filter()
x dplyr::lag()    masks stats::lag()
i Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors

Attache Paket: 'shiny'


Die folgenden Objekte sind maskiert von 'package:DT':

    dataTableOutput, renderDataTable




In [3]:
# Load the data
indo_ff <- st_read("combined_indonesia_all_years.shp")

Reading layer `combined_indonesia_all_years' from data source 
  `/Users/lokeshpandey/Library/CloudStorage/OneDrive-stud.uni-goettingen.de/Thesis/Data/ESRI/Extracted_yearly_IND/combined_indonesia_all_years.shp' 
  using driver `ESRI Shapefile'
Simple feature collection with 671360 features and 7 fields
Geometry type: MULTIPOLYGON
Dimension:     XY
Bounding box:  xmin: 95.07129 ymin: -10.82617 xmax: 141.1304 ymax: 5.828613
Geodetic CRS:  WGS 84


### Exploring the number of observation, so and so forth

In [ ]:
# Quickly check the plot

plot(st_geometry(indo_ff))

In [7]:
# Dimension
dim(indo_ff)

[1] 671360      8

671360 observations of fire over the period of 2000 till 2018 in Indonesia. Interesting.